docs/interop.md

```diff
@@ -51,18 +47,23 @@ func (t *Transaction) Claim(wallet *token.OwnerWallet, tok *token2.UnspentToken,
 func (t *Transaction) Reclaim(wallet *token.OwnerWallet, tok *token2.UnspentToken) error
 ```
 
-The interop `Wallet` service, located in `token/services/interop/wallet.go`, supports listing expired tokens, whose deadline have passed, and listing tokens with a desired matching preimage.
+The interop `Wallet` service, located under `token/services/interop/`, supports listing tokens with a desired matching preimage, and listing expired tokens, whose deadline have passed.
 
 In addition, the interop signer and verifier services are script specific, for example in the HTLC case the preimage is part of the signed message.
 
+Finally, the interoperability services which are responsible for assembling the token transaction and managing its lifecycle are the same as the [`Token Transaction Services`](./services.md).
+They are located in `token/services/interop`.
+
+
 ## Driver adjustments 
 
 The `FabToken` and `ZKAT DLog` drivers support also interoperability, and more specifically, the drivers support HTLC.
 
 In addition to the regular validation process, their `validator` ensures that in the lock and claim cases the deadline has not expired, and in the reclaim case the deadline has passed.    
```

Comment:
the `validator` in fabtoken and zkat dlog can be enhanced with extra validators to accommodate additional validation rules. In particular, to support atomic exchanges,  they take a validator that ensures HTLC conditions are met. That is, the deadline has not passed in the case of  lock,  that a claim was initiated by the recipient before the expiration of the deadline and carries the pre-image matching the hash, and the a reclaim is initiated by the sender after the deadline has passed.